src/container/rating/Rating.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./Rating.css";
+
+const Rating = ( {children} ) => {
+    let rating;
+
+    if (children < 0.5) {
+        rating = "☆☆☆☆☆";
+    } else if (children >= 0.5 && children < 1.5) {
+        rating = "★☆☆☆☆";
+    } else if (children >= 1.5 && children < 2.5) {
+        rating = "★★☆☆☆";
+    } else if (children >= 2.5 && children < 3.5) {
+        rating = "★★★☆☆";
+    } else if (children >= 3.5 && children < 4.5) {
+        rating = "★★★★☆";
+    } else if (children >= 4.5) {
+        rating = "★★★★★";
+    };
+
+    return <div className="rating">{rating}</div>;
```

Comment:
uma dica aqui seria usar o um `Math.round()` e algum loop, como um `for` ou um `map`, 

De resto ficou mto bom!